openmp/libompd/src/omp-icv.cpp

```diff
@@ -971,7 +974,7 @@ ompd_is_implicit(ompd_task_handle_t *task_handle, /* IN: OpenMP task handle*/
   return ret;
 } 
 
-static ompd_rc_t
+ompd_rc_t
```

Comment:
The purpose of static was to avoid this internal function from getting exported. I understand that you want to call the function from the other compilation unit, so you need to remove the static. 

At some point we should implement a more generic export regime (using `-Wl,-export-symbols`).